docs/neopixel.rst

```diff
@@ -0,0 +1,69 @@
+Neopixel
+****
+
+.. py:module:: neopixel
+
+The ``neopixel`` module lets you use Neopixel (WS2812) individually addressable RGB LED strips with the Microbit.    
+Note to use the ``neopixel`` module, you need to import it separately with::
+
+    import neopixel
+
+.. warning::
+    
+    Do not use the 3v connector on the Microbit to power any more than 8 Neopixels at a time.
```

Comment:
Whoops! Then even 8 pixels breaks that one... I guess they are unlikely to have them all on bright at the same time. What could possibly go wrong..